clientd3d/d3drender.c

```diff
@@ -6896,6 +6891,9 @@ void D3DRenderObjectsDraw(d3d_render_pool_new *pPool, room_type *room,
 			bottomRight.z = 0;
 			bottomRight.w = 1.0f;
 
+
+         //stretchfactor = config.viewport_stretchfactor;

```

Comment:
Delete this.  I think you can actually delete all mentions of stretchfactor in this file?

Reply:
done